### ISSUE TITLE
Bug with feminitive duke-level azerothian

### ISSUE THREAD

**Issue by EdwardEdisan** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#1128]
**Mod Version**
Frozen Throne branch
**Are you using any submods/mods? If so, which?**
No
**Please explain your issue in as much detail as possible:**
Well, this is first time, when i saw that. Wife of Anduin a Highlord

**Upload screenshots of the problem localization:**
![123](https://user-images.githubusercontent.com/24798562/73582417-be323a00-4495-11ea-93ad-ac3e17cce7d3.jpg)

**Comment by zumbak04** [comment]
English ain't my first language, so I'll refer to the **L O R E**.
https://wow.gamepedia.com/Scarlet_Highlord_Daion
Though if you suggest something better, I don't mind.

**Comment by MemeWizard** [comment]
Highlady?

**Comment by zumbak04** [comment]
I ain't sure it's better, we need English-speaker like... @DukeJeans.

**Comment by arithon** [comment]
I think, making it High Lord/Lady might be better (not taking lore into consideration)

**Comment by zumbak04** [comment]
A lot of lore characters referenced as Highlords:
https://wow.gamepedia.com/Highlord

**Comment by EdwardEdisan** [comment]
No, its impossible and really strange to pronounce to woman "Highlord". Its Highlady.

**Comment by zumbak04** [comment]
> No, its impossible and really strange to pronounce to woman "Highlord".

Well, Blizzard does it.

**Comment by EdwardEdisan** [comment]
> 
> 
> > No, its impossible and really strange to pronounce to woman "Highlord".
> 
> Well, Blizzard does it.
> https://wow.gamepedia.com/Highlord_(title)

But in this case wife of Anduin not a paladin.

**Comment by zumbak04** [comment]
> But in this case wife of Anduin not a paladin.

Highlord is used by ogres, demons, secular humans etc.
https://wow.gamepedia.com/Highlord

**Comment by rhotus** [comment]
"Highlord" doesn't actually exist in English, but "overlord" and "lord mayor" do, and both are used by women as well as men. It makes sense to have "highlord" as a female title.

**Comment by cal0497** [comment]
Personally the female version of highlord should be Lady. But it's only an opinion. Could just change it to Queen.

**Comment by zumbak04** [comment]
> Personally the female version of highlord should be Lady. But it's only an opinion. Could just change it to Queen.

@cal0497 these titles are taken. Lord and Lady are count tier titles. Queen is king tier title.

**Comment by arithon** [comment]
Ok, any decision we can strike on this topic?

**Comment by zumbak04** [comment]
> Ok, any decision we can strike on this topic?

Meeeh, don't think so. Looks like we didn't agree on this.
